src/reference/asciidoc/transformer.adoc

```diff
@@ -83,12 +143,47 @@ Spring Integration provides a few transformer implementations.
 
 ===== Object-to-String Transformer
 
-Because it is fairly common to use the `toString()` representation of an `Object`, Spring Integration provides an `ObjectToStringTransformer` whose output is a `Message` with a String `payload`.
+Because it is fairly common to use the `toString()` representation of an `Object`, Spring Integration provides an `ObjectToStringTransformer` (see also `Transformers` factory) whose output is a `Message` with a String `payload`.
```

Comment:
```suggestion
Because it is fairly common to use the `toString()` representation of an `Object`, Spring Integration provides an `ObjectToStringTransformer` (see also the `Transformers` factory) where the output is a `Message` with a String `payload`.
```